angular/core/services/session.coffee

```diff
@@ -19,6 +19,13 @@ angular.module('loomioApp').factory 'Session', ($rootScope, $location, $translat
 
     user
 
+  setLocale: (locale) ->
+    $translate.use(locale)
+    lc_locale = locale.toLowerCase()
+    fetch("/client/moment_locales/#{lc_locale}.js").then((resp) -> resp.text()).then (data) ->
+      eval(data)
```

Comment:
@gdpelican have a think about if you're ok with this eval statement. I think it's safe, but would like you to double check.